polaris.shopify.com/src/components/Examples/Examples.tsx

```diff
@@ -74,7 +112,7 @@ const Examples = (props: Props) => {
                     height={iframeHeight}
                     width="100%"
                     onLoad={handleExampleLoad}
-                    ref={iframeRef}
```

Comment:
I found the refs approach to be flaky. Maybe because the ref changes due to the Tabs. Using the `id` to find the iframe consistency works.